.github/workflows/test-gh.yml

```
@@ -65,6 +65,9 @@ jobs:
         wget -O- https://github.com/google/ko/releases/download/v0.8.0/ko_0.8.0_Darwin_x86_64.tar.gz > /tmp/ko.tgz
         tar xzvf /tmp/ko.tgz -C /tmp/bin
 
+        wget -O- https://github.com/bazelbuild/bazel/releases/download/4.2.0/bazel-4.2.0-darwin-x86_64 > /tmp/bin/bazel
```

```suggestion
        wget -O- -nv https://github.com/bazelbuild/bazel/releases/download/4.2.0/bazel-4.2.0-darwin-x86_64 > /tmp/bin/bazel
```

In a parallel PR (#153), we're attempting to reduce the noise that comes from the progress output from `wget`. Mind including the `-nv` flag, here?

---

🙏🏻